src/ts/content/extensions/tags.ts

```diff
@@ -40,9 +40,10 @@ onFormRender((form, forEachElement, onSave, offSave) => {
 			textAreaId: tagsTextAreaId,
 			transform: (tags: string[]) => tags.join(", "),
 			delimiter: ", ",
+			requiresLogged: true,
 		});
 
-		const {showFinder: showAncestors, hideFinder: hideAncestors} = appendFinder<string[]>({
+		insertFinder<string[]>({
```

Comment:
oh cool so now we just call this instead of unpacking stuff and putting them back into `onLogged`?

Reply:
yesssss